README.md

```diff
@@ -38,6 +38,11 @@ The path (`/output/path/`) must be absolute.
 - Change the text orientation using the `-or` parameter
 - Specify text color range using `-tc '#000000,#FFFFFF'`, please note that the quotes are **necessary**
 - Add support for Simplified and Traditional Chinese
+- Generation Strings with regular expressions `--regex`
+- Specify Image folder for background `--images_dir `
```

Comment:
Why is the fonts argument named `fonts`, but the images one has a suffix `_dir`?

Reply:
You are right, just --images